indico/cli/setup.py

```diff
@@ -402,12 +413,14 @@ def _get_noreply_email(email):
                                      help='When Indico sends email notifications they are usually sent from the '
                                           'noreply email.')
 
-    def _prompt_smtp_data(self):
+    def _prompt_smtp_data(self, dev=False):
```

Comment:
Should we even prompt for this in dev mode instead of just setting 127.0.0.1:1025?

Reply:
Should we assume people will always use maildump?

Reply:
for development it might be a good idea instead of having people put their gmail credentials in the indico config (which i think is not too uncommon, as ugly as it is :/)

Reply:
What's wrong about having a default that looks good and still allowing for some flexibility? After all, people may choose to use other than maildump...